client/src/reducers/index.js

```diff
@@ -1,9 +1,11 @@
 import { combineReducers } from 'redux'
 import { reducer as toastrReducer } from 'react-redux-toastr'
 import auth from './auth'
+import newsFeed from './newsFeed'
 
 export default combineReducers({
   auth,
+  newsFeed,
```

Comment:
I would rather call this reducer something like  "posts"

Reply:
as you wish :) Done

Reply:
Danke!